### DISCUSSION TITLE
[Unstable] AI refuses to do any commands on abandoned outposts, including finding a killing targets.

### DISCUSSION THREAD

**Discussion by PanmanS** [FakeFishGames/Barotrauma#13663] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Despite my bots being well equipped with weapons and stuff, AI fiercely refuses to do ANYTHING on abadoned outposts. It would be nice if I could order a death squad of my bots to kill everything there or engineer to repair junctions there!

### Reproduction steps

1. start abandoned outpost mission in any way with bots
2. equip bot with gear you like and order it to attack introdures while in docking module of abadoned outpost
3. AI will refuse you.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.2.13.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I think this is actually the intended behavior: the bots only consider the items in their own submarine when you tell them to maintain or repair things. The "fight intruders" order makes them fight enemies that enter the submarine, if you want them to instead attack enemies in the outpost, you need to issue the "assault enemy" order.

Even though I agree it would sometimes be useful to order the bots to repair and operate things in outposts, I would consider this more of a feature request than a bug report, so I'll move it to "ideas and suggestions".

**Comment by G-M-twostay** [comment]
doesn't this also happens in stable and normal outposts? when bots are in outposts the only command they listen to is movement commands.
